components/raftstore/src/store/entry_storage.rs

```diff
@@ -958,6 +958,26 @@ impl<EK: KvEngine, ER: RaftEngine> EntryStorage<EK, ER> {
         }
     }
 
+    #[inline]
+    pub fn truncate_index(&self) -> u64 {
```

Comment:
I think there is already `truncated_index` and `truncated_term` methods.

Reply:
get_truncate_index 
get_truncate_term

Reply:
yes，I have found the similar func